2018-03-29 09:22:46 Error: [MissingActionException] Action AppController::index() could not be found.
Exception Attributes: array (
  'controller' => 'AppController',
  'action' => 'index',
)
Request URL: /transport_app_ver2_1/app/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#1 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(AppController), Object(CakeRequest), Object(CakeResponse))
#2 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 /var/www/html/transport_app_ver2_1/app/index.php(19): require('/var/www/html/t...')
#4 {main}
2018-03-29 09:22:53 Error: [MissingActionException] Action AppController::webroot() could not be found.
Exception Attributes: array (
  'controller' => 'AppController',
  'action' => 'webroot',
)
Request URL: /transport_app_ver2_1/app/webroot/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#1 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(AppController), Object(CakeRequest), Object(CakeResponse))
#2 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2018-03-30 17:05:32 Error: [MissingControllerException] Controller class RequestDetailsController could not be found.
Exception Attributes: array (
  'class' => 'RequestDetailsController',
  'plugin' => NULL,
)
Request URL: /transport_app_ver2_1/request_details/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2018-03-30 17:06:14 Error: [MissingTableException] Table requestdetails for model Requestdetail was not found in datasource default.
Exception Attributes: array (
  'table' => 'requestdetails',
  'class' => 'Requestdetail',
  'ds' => 'default',
)
Request URL: /transport_app_ver2_1/request_details/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(3498): Model->setSource('requestdetails')
#1 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(1355): Model->getDataSource()
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(1447): Model->schema()
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(1434): Model->hasField('title', false)
#4 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(880): Model->hasField(Array)
#5 /var/www/html/transport_app_ver2_1/lib/Cake/Controller/Scaffold.php(131): Model->__get('displayField')
#6 /var/www/html/transport_app_ver2_1/lib/Cake/Controller/Controller.php(535): Scaffold->__construct(Object(RequestdetailsController), Object(CakeRequest))
#7 /var/www/html/transport_app_ver2_1/lib/Cake/Controller/Controller.php(494): Controller->_getScaffold(Object(CakeRequest))
#8 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#9 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(RequestdetailsController), Object(CakeRequest), Object(CakeResponse))
#10 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2018-03-30 17:06:50 Error: [MissingTableException] Table requestdetails for model Requestdetail was not found in datasource default.
Exception Attributes: array (
  'table' => 'requestdetails',
  'class' => 'Requestdetail',
  'ds' => 'default',
)
Request URL: /transport_app_ver2_1/request_details/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(3498): Model->setSource('requestdetails')
#1 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(1355): Model->getDataSource()
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(1447): Model->schema()
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(1434): Model->hasField('title', false)
#4 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(880): Model->hasField(Array)
#5 /var/www/html/transport_app_ver2_1/lib/Cake/Controller/Scaffold.php(131): Model->__get('displayField')
#6 /var/www/html/transport_app_ver2_1/lib/Cake/Controller/Controller.php(535): Scaffold->__construct(Object(RequestdetailsController), Object(CakeRequest))
#7 /var/www/html/transport_app_ver2_1/lib/Cake/Controller/Controller.php(494): Controller->_getScaffold(Object(CakeRequest))
#8 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#9 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(RequestdetailsController), Object(CakeRequest), Object(CakeResponse))
#10 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2018-03-30 17:41:37 Error: [MissingActionException] Action UsersController::index() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'index',
)
Request URL: /transport_app_ver2_1/users/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#1 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#2 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2018-04-03 18:39:19 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'id' in order clause is ambiguous
Request URL: /transport_app_ver2_1/request_details/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Datasource/DboSource.php(458): PDOStatement->execute(Array)
#1 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Datasource/DboSource.php(424): DboSource->_execute('SELECT `Request...', Array)
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Datasource/DboSource.php(665): DboSource->execute('SELECT `Request...', Array, Array)
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Datasource/DboSource.php(1077): DboSource->fetchAll('SELECT `Request...', false)
#4 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(2902): DboSource->read(Object(RequestDetail), Array)
#5 /var/www/html/transport_app_ver2_1/lib/Cake/Model/Model.php(2874): Model->_readDataSource('all', Array)
#6 /var/www/html/transport_app_ver2_1/app/Controller/RequestDetailsController.php(12): Model->find('all', Array)
#7 [internal function]: RequestDetailsController->index()
#8 /var/www/html/transport_app_ver2_1/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(RequestDetailsController), Array)
#9 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#10 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(RequestDetailsController), Object(CakeRequest), Object(CakeResponse))
#11 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2018-04-04 11:45:17 Error: Fatal Error (4): syntax error, unexpected '<' in [/var/www/html/transport_app_ver2_1/app/View/Users/index.ctp, line 15]
2018-04-04 11:45:17 Error: [FatalErrorException] syntax error, unexpected '<'
Request URL: /transport_app_ver2_1/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/html/t...', 15)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/html/t...', 15, Array)
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(929): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/html/t...', 15, Array)
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2018-04-04 12:44:23 Error: Fatal Error (4): syntax error, unexpected '}' in [/var/www/html/transport_app_ver2_1/app/Controller/UsersController.php, line 23]
2018-04-04 12:44:23 Error: [FatalErrorException] syntax error, unexpected '}'
Request URL: /transport_app_ver2_1/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/html/t...', 23)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/html/t...', 23, Array)
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(929): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/html/t...', 23, Array)
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2018-04-04 12:51:46 Error: [MissingHelperException] Helper class testHelper could not be found.
Exception Attributes: array (
  'class' => 'testHelper',
  'plugin' => false,
)
Request URL: /transport_app_ver2_1/
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/View/HelperCollection.php(63): HelperCollection->load('test')
#1 /var/www/html/transport_app_ver2_1/lib/Cake/View/View.php(818): HelperCollection->__isset('test')
#2 /var/www/html/transport_app_ver2_1/app/View/Users/index.ctp(33): View->__get('test')
#3 /var/www/html/transport_app_ver2_1/lib/Cake/View/View.php(929): include('/var/www/html/t...')
#4 /var/www/html/transport_app_ver2_1/lib/Cake/View/View.php(891): View->_evaluate('/var/www/html/t...', Array)
#5 /var/www/html/transport_app_ver2_1/lib/Cake/View/View.php(460): View->_render('/var/www/html/t...')
#6 /var/www/html/transport_app_ver2_1/lib/Cake/Controller/Controller.php(952): View->render(NULL, NULL)
#7 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(192): Controller->render()
#8 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#9 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2018-04-04 13:07:31 Error: [MissingControllerException] Controller class PostsController could not be found.
Exception Attributes: array (
  'class' => 'PostsController',
  'plugin' => NULL,
)
Request URL: /transport_app_ver2_1/posts/request_detail
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2018-04-04 13:08:09 Error: [MissingControllerException] Controller class PostsController could not be found.
Exception Attributes: array (
  'class' => 'PostsController',
  'plugin' => NULL,
)
Request URL: /transport_app_ver2_1/posts/request_detail
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2018-04-04 13:08:24 Error: [MissingActionException] Action RequestDetailsController::request_detail() could not be found.
Exception Attributes: array (
  'controller' => 'RequestDetailsController',
  'action' => 'request_detail',
)
Request URL: /transport_app_ver2_1/request_details/request_detail
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#1 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(RequestDetailsController), Object(CakeRequest), Object(CakeResponse))
#2 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2018-04-04 13:08:49 Error: [MissingActionException] Action RequestDetailsController::request_detail() could not be found.
Exception Attributes: array (
  'controller' => 'RequestDetailsController',
  'action' => 'request_detail',
)
Request URL: /transport_app_ver2_1/requestdetails/request_detail
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#1 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(RequestDetailsController), Object(CakeRequest), Object(CakeResponse))
#2 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2018-04-04 16:25:32 Error: Fatal Error (64): Cannot use [] for reading in [/var/www/html/transport_app_ver2_1/app/View/RequestDetails/index.ctp, line 26]
2018-04-04 16:25:32 Error: [FatalErrorException] Cannot use [] for reading
Request URL: /transport_app_ver2_1/request_details
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(64, 'Cannot use [] f...', '/var/www/html/t...', 26)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot use [] f...', '/var/www/html/t...', 26, Array)
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(929): call_user_func('ErrorHandler::h...', 64, 'Cannot use [] f...', '/var/www/html/t...', 26, Array)
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2018-04-04 16:25:42 Error: Fatal Error (64): Cannot use [] for reading in [/var/www/html/transport_app_ver2_1/app/View/RequestDetails/index.ctp, line 26]
2018-04-04 16:25:42 Error: [FatalErrorException] Cannot use [] for reading
Request URL: /transport_app_ver2_1/request_details
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(64, 'Cannot use [] f...', '/var/www/html/t...', 26)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot use [] f...', '/var/www/html/t...', 26, Array)
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(929): call_user_func('ErrorHandler::h...', 64, 'Cannot use [] f...', '/var/www/html/t...', 26, Array)
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2018-04-04 17:54:48 Error: [MissingControllerException] Controller class View.ctpController could not be found.
Exception Attributes: array (
  'class' => 'View.ctpController',
  'plugin' => NULL,
)
Request URL: /transport_app_ver2_1/view.ctp
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2018-04-04 17:55:46 Error: [MissingActionException] Action UsersController::view.ctp() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'view.ctp',
)
Request URL: /transport_app_ver2_1/users/view.ctp
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#1 /var/www/html/transport_app_ver2_1/lib/Cake/Routing/Dispatcher.php(160): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#2 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2018-04-04 17:56:08 Error: Fatal Error (1): Call to a member function find() on null in [/var/www/html/transport_app_ver2_1/app/Controller/UsersController.php, line 32]
2018-04-04 17:56:08 Error: [FatalErrorException] Call to a member function find() on null
Request URL: /transport_app_ver2_1/users/view
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/html/t...', 32)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/html/t...', 32, Array)
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/html/t...', 32, Array)
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2018-04-04 18:45:09 Error: Fatal Error (4): syntax error, unexpected '=' in [/var/www/html/transport_app_ver2_1/app/View/RequestDetails/index.ctp, line 39]
2018-04-04 18:45:09 Error: [FatalErrorException] syntax error, unexpected '='
Request URL: /transport_app_ver2_1/request_details
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/var/www/html/t...', 39)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/var/www/html/t...', 39, Array)
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(929): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/var/www/html/t...', 39, Array)
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2018-04-04 18:46:15 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /transport_app_ver2_1/add
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2018-04-04 18:47:37 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /transport_app_ver2_1/add
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2018-04-04 18:48:21 Error: Fatal Error (1): Call to a member function save() on null in [/var/www/html/transport_app_ver2_1/app/Controller/RequestdetailsController.php, line 41]
2018-04-04 18:48:21 Error: [FatalErrorException] Call to a member function save() on null
Request URL: /transport_app_ver2_1/requestdetails/add
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/html/t...', 41)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/html/t...', 41, Array)
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/html/t...', 41, Array)
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2018-04-04 18:49:10 Error: Fatal Error (1): Call to a member function save() on null in [/var/www/html/transport_app_ver2_1/app/Controller/RequestdetailsController.php, line 41]
2018-04-04 18:49:10 Error: [FatalErrorException] Call to a member function save() on null
Request URL: /transport_app_ver2_1/requestdetails/add
Stack Trace:
#0 /var/www/html/transport_app_ver2_1/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/html/t...', 41)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/html/t...', 41, Array)
#2 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/html/t...', 41, Array)
#3 /var/www/html/transport_app_ver2_1/lib/Cake/Core/App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
